test/autests/gold_tests/http2_frames/http2_frames.yaml

```diff
@@ -23,6 +23,7 @@ sessions:
             - [Content-Length, '11']
             - [uuid, 1]
       - DATA:
+          delay: 1s
```

Comment:
It would be good to verify the delay.

When I added delay, I added the delay.test.py test which has a script that verifies things took a certain amount of time. That can be helpful to test with these directives as well.

Reply:
Ah,  nevermind. You did verify this here:
https://github.com/yahoo/proxy-verifier/pull/251/files#diff-1b6b104f5799873ebc4f958cd837c67ddc215a263c3d026e089d4935fab3eb6fR38